docs/effects/api.md

```diff
@@ -5,35 +5,29 @@
 NgModule for @ngrx/effects.
 
 ### forRoot
+
 Registers internal @ngrx/effects services to run in your application. This is required once in your root NgModule.
 
 Usage:
+
 ```ts
 @NgModule({
-  imports: [
-    EffectsModule.forRoot([
-      FirstEffectsClass,
-      SecondEffectsClass,
-    ])
-  ]
+  imports: [EffectsModule.forRoot([FirstEffectsClass, SecondEffectsClass])],
 })
-export class AppModule { }
+export class AppModule {}
 ```
 
 ### forFeature
+
 Registers @ngrx/effects services to run with your feature modules.
 
 **Note**: Running an effects class multiple times, either by `forRoot()` or `forFeature()`, (for example via different lazy loaded modules) will not cause Effects to run multiple times. There is no functional difference between effects loaded by `forRoot()` and `forFeature()`; the important difference between the functions is that `forRoot()` sets up the providers required for effects.
 
 Usage:
+
 ```ts
 @NgModule({
-  imports: [
-    EffectsModule.forFeature([
-      SomeEffectsClass,
-      AnotherEffectsClass,
-    ])
-  ]
+  imports: [EffectsModule.forFeature([SomeEffectsClass, AnotherEffectsClass])],
```

Comment:
Previous formatting was way more readable. This is read me after all.

Reply:
This was because of husky and prettier auto run after I commit

Reply:
If needed, this you can use `prettier-ignore` to ignore a line/file/...

Reply:
I think changing format setup is not the main purpose of this PR, I just used the config that has been set. If brandon or mike suggest it I will change it accordingly.